src/main/java/org/apache/maven/buildcache/BuildCacheMojosExecutionStrategy.java

```diff
@@ -96,6 +97,7 @@ public void execute( List<MojoExecution> mojoExecutions,
         // execute clean bound goals before restoring to not interfere/slowdown clean
         CacheState cacheState = DISABLED;
         CacheResult result = CacheResult.empty();
+        boolean skipCacheLookup = cacheConfig.isSkipLookup() || MavenProjectInput.shouldSkipCacheLookup( project );
```

Comment:
regarding the naming - lets call it skipCache. Gradle has similar parameter to force rebuild named `--no-build-cache`. from command line and pom perspective -DskipBuildCache is more descriptive

Reply:
@AlexanderAshitkin renamed